kernel/src/main/java/org/kframework/unparser/Formatter.java

```diff
@@ -96,7 +85,7 @@ public static void format(Term term, Indenter indenter, ColorSetting colorize) {
                             Term inner = tc.get(nt);
                             boolean assoc = false;
                             if (inner instanceof TermCons) {
-                                TermCons innerTc = (TermCons)inner;
+                                TermCons innerTc = (TermCons) inner;
```

Comment:
This can be changed to something like this:
```java
if (inner instanceof TermCons innerTc) {
...
}
```
In fact, there are a lot of places where this can be done. It might deserve another PR.

Reply:
Yes, I'm going to follow up and do this automatically.